qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -2,6 +2,18 @@
     only Fedora, RHEL, Windows
     no Fedora.8, Fedora.9, Fedora.10, Fedora.11, Fedora.12, Fedora.13, Fedora.14, Fedora.15
     type = qemu_guest_agent
+    virtio_win_media_type = iso
+    driver_name_serial = vioser
+    device_hwid_serial = '"PCI\VEN_1AF4&DEV_1003" "PCI\VEN_1AF4&DEV_1043"'
+    cmd_check_serial = 'driverquery /V |findstr /i "virtioserial"'
+    cert_files = ""
+    devcon_dirname = "win7_"
+    i386:
+        devcon_dirname += "x86"
+    x86_64:
+        devcon_dirname += "amd64"
+    devcon_path = "WIN_UTILS:\devcon\${devcon_dirname}\devcon.exe"
+    cmd_serial_driver_install = '${devcon_path} updateni %s %s'
```

Comment:
You might write here `check_vioser = no`, and at Line37 after `Windows:` write `check_vioser = yes`, and move the above lines down to `Windows` block too, that will take affect on all test cases.